robottelo/ui/login.py

```diff
@@ -38,8 +38,25 @@ def logout(self):
         self.wait_for_ajax()
 
     def is_logged(self):
-        """Checks whether an user is logged"""
-        if self.wait_until_element(locators["login.gravatar"]):
+        """Checks whether an user is logged by validating the session cookie"""
+        cookies = self.browser.get_cookies()
+        wd_cookies = self.browser.get_cookies()
+        cookies = {}
+        # reformatting the cookies for requests.get request
+        for cookie in wd_cookies:
+            cookies[cookie['name']] = cookie['value']
+        # extract the root from the URL
+        url_root = self.browser.current_url.split('/')[2]
+
+        r = requests.get(
+            'https://'+url_root,
```

Comment:
get_server_url should give you the right url, without the need of prepending 'https://'